gatsby/src/docs/workflow/search/index.mdx

```diff
@@ -32,6 +32,24 @@ The tokens `is:resolved` and `user.username:"Jane Doe"` are standard search toke
 
 The token `example error` is utilizing the optional raw search and is passed as part of the issue search query (which uses a CONTAINS match similar to SQL). When using the optional raw search, you can provide _one_ string, and the query uses that entire string.
 
+### Using `OR` and `AND`
+
+NOTE: `OR` and `AND` search conditions are only available for [Discover](/performance-monitoring/discover-queries/), [Performance](/performance-monitoring/performance/), and [Metric Alerts](/workflow/alerts-notifications/alerts/#metric-alerts).
+
+Use `OR` and `AND` between tokens, and use parentheses `()` to group conditions. `AND` can also be used between non-aggregates and aggregates. However, `OR` cannot. 
```

Comment:
@evanh are aggregates searches only applicable to discover?

Reply:
@adhiraj They're applicable to Discover, Performance, and Performance Metric Alerts.

Reply:
Adi and I were thinking of defining the difference between an aggregate and a non-aggregate. How's this:

Aggregate functions: groups data on numerical scales (for example, count():>100)
Non-aggregate functions: groups data based on specific attributes (for example, user.username:Jane)

Reply:
I think that works.

Reply:
@evanh do you think we can improve on the definitions of aggregate vs non-aggregate functions? I drafted some definitions above.